seedlet/assets/sass/blocks/utilities/_style.scss

```diff
@@ -1,5 +1,13 @@
 /* Block Alignments */
 
+/**
+ * These selectors set the default max width for content appearing inside a post or page. 
+ */
+.entry-content > *:not(.alignwide):not(.alignfull):not(.wp-block-separator):not(.entry-attachment):not(.woocommerce),
```

Comment:
I removed `:not(.alignleft):not(.alignright)` from these selectors. Otherwise, blocks aligned left or right will overflow from the main column if their content is wider than the default width.